### PR TITLE
Ballistics - Move dispersion to optional

### PR DESCRIPTION
**Pull request by Laid3acK**
according with https://github.com/acemod/ACE3/pull/7532#issuecomment-589994082

### CODE REVIEW

File: optionals/compat_ace_dispersion/script_component.hpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#define COMPONENT compat_ace_dispersion
```
Comment (jonpas) [inline optionals/compat_ace_dispersion/script_component.hpp]: `realisticdispersion` maybe? Mimics `realisticnames`.

`ace` is already there as a prefix. It's not really a compatibility.

File: optionals/compat_ace_dispersion/script_component.hpp [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+#define COMPONENT_BEAUTIFIED ACE3 Dispersion Compatibility
```
Comment (jonpas) [inline optionals/compat_ace_dispersion/script_component.hpp]: It's not really compatibility, more that it provides realistic values.

File: optionals/compat_ace_dispersion/$PBOPREFIX$ [hunk 3]
```diff
@@ -0,0 +1 @@
+z\ace\addons\compat_ace_dispersion
```
Comment (jonpas) [inline optionals/compat_ace_dispersion/$PBOPREFIX$]:
```suggestion
z\ace\addons\compat_ace_dispersion

```

File: optionals/compat_ace_dispersion/script_component.hpp (hunk 1 above)
Comment (Laid3acK) [reply optionals/compat_ace_dispersion/script_component.hpp]: Nothing realistic with these dispersions, except for casuals.
Rename that as you want.

File: optionals/compat_ace_dispersion/script_component.hpp (hunk 2 above)
Comment (Laid3acK) [reply optionals/compat_ace_dispersion/script_component.hpp]: Same.

File: optionals/compat_ace_dispersion/script_component.hpp (hunk 1 above)
Comment (jonpas) [reply optionals/compat_ace_dispersion/script_component.hpp]: Wasn't your plan to improve them? I'd like to avoid renaming.

Comment (Laid3acK) [reply optionals/compat_ace_dispersion/script_component.hpp]: Do it please Jonpas, no problem.
To bad connexion at this moment.

**Pull request closed without merging**